modules/vba_documents/lib/vba_documents/upload_validator.rb

```diff
@@ -76,6 +76,7 @@ def validate_documents(parts, pdf_validator_options = VBADocuments::DocumentRequ
 
     def perfect_metadata(model, parts, timestamp)
       metadata = JSON.parse(parts['metadata'])
+      metadata['fileNumber'] = metadata['fileNumber'].strip if metadata.key?('fileNumber')
```

Comment:
The `if metadata.key?('fileNumber')` isn't needed (see previous review comment) but doesn't harm anything here.